### PR TITLE
[OSDOCS-2971]: use GCP Marketplace image via machineset

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.8+

Issue:
[OSDOCS-2971](https://issues.redhat.com//browse/OSDOCS-2971)

Link to docs preview:
[Sample YAML for a compute machine set custom resource on GCP](https://52429--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-gcp.html#machineset-yaml-gcp_creating-machineset-gcp)

QE review:
- [ ] QE has approved this change.

Additional information:
- This will need to go into 4.12 first, then coordinate with @mjpytlak on cherrypicks
- Machine mgmt component to accompany install work in #51678
- Also took the opportunity to move the `taints` param to the bottom to make the YAML file easier to maintain
- Removed duplicate callouts from the YAML block to see if this resolves the formatting issue on the portal we are seeing in [CCS-6623](https://issues.redhat.com//browse/CCS-6623)

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@patrickdillon PTAL (I'll also ping the cloud team for a YAML sanity check)

File: modules/machineset-yaml-gcp.adoc
Comment (jeana-redhat) [inline modules/machineset-yaml-gcp.adoc:123]: Note: this is the actual functional content change for Marketplace images

**Comment by mjpytlak** [comment]
@jeana-redhat hold off on merging this as I have to double back on https://github.com/openshift/openshift-docs/pull/51678 to add the images for OKE and OPP. I will reference the doc PR as soon as it is up. Likely tomorrow. Of course, here are the references in the KCS
# For OCP
projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145

# For OPP
projects/redhat-marketplace-public/global/images/redhat-coreos-opp-48-x86-64-202206140145

# For OKE
projects/redhat-marketplace-public/global/images/redhat-coreos-oke-48-x86-64-202206140145

**Comment by jeana-redhat** [comment]
close/reopen to get a new build now that Mike's new OPP and OKE attributes are merged into `main`

**Comment by jeana-redhat** [comment]
@sunzhaohua2, @huali9, or @miyadav this is ready for verification, PTAL. There's a lot of reformatting, but the new content is [here](https://github.com/openshift/openshift-docs/pull/52429/files#diff-4509739346813f02b9acbd27438315106f823c6a9ac072dac792f6560707dc06R123-R129)

**Comment by miyadav** [comment]
Thanks @jeana-redhat , I am not sure if images will be of the form - `ocp-48` or `ocp-412` ? 
If those correct URLs , LGTM , but need to confirm them.

**Comment by jeana-redhat** [comment]
Thanks, @miyadav! The images are taken from #52617, which has approval from @jianli-wei - maybe they can confirm

**Review by adellape — APPROVED** [review]

**Comment by jeana-redhat** [comment]
Force pushed to force rebuild before merging, no change.

**Pull request merged**
